test/acceptance/features/claimFromApplicationNamespace.feature

```diff
@@ -0,0 +1,284 @@
+          serviceEndpointDefinition:
+            - name: host
+              value: mydavphost.io
+            - name: port
+              value: "5432"
+            - name: user
+              value: davp
+            - name: password
+              value: quedicelagente
+            - name: database
+              value: davpdata
+          sla: L3
+          """
+        And On Primaza Cluster "main", RegisteredService "primaza-rsdb" state will eventually move to "Available"
+        And On Worker Cluster "worker", Resource is created
+        """
+        apiVersion: primaza.io/v1alpha1
+        kind: ServiceClaim
+        metadata:
+          name: sc-test
+          namespace: applications
+        spec:
+          serviceClassIdentity:
+          - name: type
+            value: psqlserver
+          - name: provider
+            value: aws
+          serviceEndpointDefinitionKeys:
+          - host
+          - port
+          - user
+          - password
+          - database
+          environmentTag: stage
+          application:
+            kind: Deployment
+            apiVersion: apps/v1
+            selector:
+              matchLabels:
+                a: b
+                c: d
+        """
+        And On Worker Cluster "worker", the status of ServiceClaim "sc-test" is "Resolved"
+        And On Primaza Cluster "main", RegisteredService "primaza-rsdb" state will eventually move to "Claimed"
+        And On Primaza Cluster "main", ServiceCatalog "stage" will not contain RegisteredService "primaza-rsdb"
+        And On Worker Cluster "worker", the secret "sc-test" in namespace "applications" has the key "type" with value "psqlserver"
+        And On Worker Cluster "worker", Resource is updated
+        """
+        apiVersion: primaza.io/v1alpha1
+        kind: ServiceClaim
+        metadata:
+          name: sc-test
+          namespace: applications
+        spec:
+          serviceClassIdentity:
+          - name: type
+            value: mysql
+          - name: provider
+            value: aws
+          serviceEndpointDefinitionKeys:
+          - host
+          - port
+          - user
+          - password
+          - database
+          environmentTag: stage
+          application:
+            kind: Deployment
+            apiVersion: apps/v1
+            selector:
+              matchLabels:
+                a: b
+                c: d
+        """
+        And On Worker Cluster "worker", the status of ServiceClaim "sc-test" is "Resolved"
+        And On Primaza Cluster "main", RegisteredService "primaza-rsdb" state will eventually move to "Claimed"
+        And On Primaza Cluster "main", ServiceCatalog "stage" will not contain RegisteredService "primaza-rsdb"
+        And jsonpath ".spec.serviceClassIdentity[0]" on "serviceclaims.primaza.io/sc-test:primaza-system" in cluster main is "{"name":"type","value":"mysql"}"
+
+    Scenario: Delete claim with label selector from an application namespace
```

Comment:
`        Given Primaza Cluster "main" is running
        And Worker Cluster "worker" for "main" is running
        And Clusters "main" and "worker" can communicate
        And On Primaza Cluster "main", Worker "worker"'s ClusterContext secret "primaza-kw" is published
        And On Worker Cluster "worker", application namespace "applications" exists
        And Primaza cluster's "main" kubeconfig is available on "worker" in namespace "applications"
        And On Primaza Cluster "main", Resource is created
        """
        apiVersion: primaza.io/v1alpha1
        kind: ClusterEnvironment
        metadata:
            name: worker
            namespace: primaza-system
        spec:
            environmentName: stage
            clusterContextSecret: primaza-kw
            applicationNamespaces:
            - applications
        """`

All this seems common. Let's move this to background

Reply:
Done